Pipelines/publish_wiki.yml

```diff
@@ -0,0 +1,53 @@
+parameters:
+- name: 'USER_NAME'
+  type: string
+- name: 'USER_EMAIL'
+  type: string
+- name: 'USER_TOKEN'
+  type: string
+
+jobs:
+- job: publish_wiki
+  timeoutInMinutes: 1
```

Comment:
the timeout should be longer, e.g. 10 or 30 mins 🤔

Reply:
The [original](https://github.com/nimblehq/github-actions-workflows/blob/dc4623e1a56ab0900ff7be4765cf467047a7877e/.github/workflows/publish_wiki.yml#L61) has 1 minute timeout. I think 5 should be enough, it's just pushing. The intension of the original 1 minutes possible be about billable usage 🤔